src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.RyuJit.cs

```diff
@@ -1504,9 +1515,94 @@ private unsafe HRESULT allocPgoInstrumentationBySchema(CORINFO_METHOD_STRUCT_* f
             throw new NotImplementedException("allocPgoInstrumentationBySchema");
         }
 
-        private HRESULT getPgoInstrumentationResults(CORINFO_METHOD_STRUCT_* ftnHnd, ref PgoInstrumentationSchema* pSchema, ref uint pCountSchemaItems, byte** pInstrumentationData)
+        private PgoSchemaElem[] getPgoInstrumentationResults(MethodDesc method)
         {
-            throw new NotImplementedException("getPgoInstrumentationResults");
+            return _compilation.ProfileData[method]?.SchemaData;
+        }
+
+        private HRESULT getPgoInstrumentationResults(CORINFO_METHOD_STRUCT_* ftnHnd, ref PgoInstrumentationSchema* pSchema, ref uint countSchemaItems, byte** pInstrumentationData)
```

Comment:
I'm moving all of this to Common in the runtime repo - we can delete all of this later.